cmd/srcd/daemon/daemon.go

```diff
@@ -166,6 +166,8 @@ func createDaemon(workdir string) docker.StartFunc {
 		if err != nil {
 			return err
 		}
+		// we run the command inside docker so slashes must be always converted to unix-style
```

Comment:
Is this right? this is run by the client, on the host, not inside a container.

Reply:
we run the command on client but the value is used inside container: https://github.com/src-d/engine/pull/355/files#diff-f65e86a43ab65d559e1ce8134b4aa922R197
or am I wrong?

Reply:
ok yes, I understood "the command" as the command calling `createDaemon()`